pkg/datapath/linux/config/config.go

```diff
@@ -718,11 +718,7 @@ func (h *HeaderfileWriter) writeTemplateConfig(fw *bufio.Writer, e datapath.Endp
 		fmt.Fprintf(fw, "#define FORCE_LOCAL_POLICY_EVAL_AT_SOURCE 1\n")
 	}
 
-	if e.RequireRouting() {
-		fmt.Fprintf(fw, "#define ENABLE_ROUTING 1\n")
-	}
-
-	if e.RequireEndpointRoute() {
+	if e.RequireEndpointRoute() || !e.RequireRouting() {
```

Comment:
Datapath side assumes `!e.RequireRouting() == e.RequireEndpointRoute()`. Could we then just delete the `RequireRouting` method?

Reply:
Do you mean just delete RequireRouting() from pkg/datapath/linux/config/config.go file alone?
or 
you wanted to delete all the references to RequireRouting() ?

Reply:
All references.